calabash-cucumber/scripts/.irbrc

```diff
@@ -47,3 +43,22 @@ extend Calabash::Cucumber::Operations
 def embed(x,y=nil,z=nil)
   puts "Screenshot at #{x}"
 end
+
+require "calabash-cucumber"
+
+def preferences
+  @preferences ||= Calabash::Cucumber::Preferences.new
```

Comment:
> fetch or get

At the moment, the entire Preferences API is doc'd private which means that we can change it.  I think this is the right API because I don't actually want to fetch or get anything with constructor.  I want the `IO` to be done when the accessors and setters are called.

>  don't think we'd be losing a lot by reading preferences from disk each time this is called rather than storing it in an ivar

`new` just creates an object; it does not set any internal state.  It does not `read` the preferences plist - for exactly reason you are making.

The accessors and setters read and write as you call them.  Having a memoized variable here is just a convenience, but maybe it is confusing?